### PR TITLE
Updated Twig template to take into account asset() function changes

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7+ |
| Fixed tickets | #4982 (partially) |

### CODE REVIEW

File: book/templating.rst [hunk 1]
```diff
@@ -1019,46 +1019,12 @@ assets won't be cached when deployed. For example, ``/images/logo.png`` might
 look like ``/images/logo.png?v2``. For more information, see the :ref:`ref-framework-assets-version`
 configuration option.
 
-.. _`book-templating-version-by-asset`:
```
Comment (xabbuh) [inline book/templating.rst]: Will we have anything version-related left in the docs when this is removed?

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -420,6 +420,8 @@ For more details, see :doc:`/cookbook/serializer`.
 templating
 ~~~~~~~~~~
 
+.. _ref-framework-assets-base-urls:
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: We need to be careful that this label is not removed when #5170 is merged.

File: reference/twig_reference.rst [hunk 3]
```diff
@@ -126,6 +122,29 @@ assets_version
 Returns the current version of the package, more information in
 :ref:`book-templating-assets`.
 
+absolute_url
+~~~~~~~~~~~~
+
```
Comment (xabbuh) [inline reference/twig_reference.rst]: We should add a `versionadded` directive here.

File: reference/twig_reference.rst [hunk 4]
```diff
@@ -126,6 +122,29 @@ assets_version
...
+    **type**: ``string``
+``packageName``
+    **type**: ``string`` | ``null`` **default**: ``null``
+
+Returns the absolute URL that corresponds to the given asset path and package.
+More information in :ref:`book-templating-assets`. For configuring the base URLs,
+:ref:`ref-framework-assets-base-urls`.
+
+The absolute URLs generated with this function ignore the asset versioning.
+Combine it with the ``assets_version()`` function to append the version number:
```
Comment (xabbuh) [inline reference/twig_reference.rst]: What about adding a separate section for the `assets_version()` function?

File: book/templating.rst [hunk 5]
```diff
@@ -1019,46 +1019,12 @@ assets won't be cached when deployed. For example, ``/images/logo.png`` might
...
-
-    .. code-block:: html+php
+.. code-block:: html+jinja
 
-        <img src="<?php echo $view['assets']->getUrl(
-            'images/logo.png',
-            null,
-            true
-        ) ?>" alt="Symfony!" />
+    <img src="{{ absolute_url(asset('images/logo.png')) }}" alt="Symfony!" />
```
Comment (WouterJ) [inline book/templating.rst]: we have to document the PHP format

Comment (stof) [inline book/templating.rst]: this is the PHP format:

```html+php
<img src="<?php echo $view['assets']->getUrl('images/logo.png') ?>" alt="Symfony!" />
```

Comment (stof) [inline book/templating.rst]: hmm no sorry. I'm missing the absolute_url part. I'm not sure we have the helper for that in the PHP templating engine though

however, the exsting doc is wrong too. Even in the deprecated asset system, there is no way to get an absolute URL in the PHP helper (the deprecated third argument is the version, not an absolute flag)

Comment (WouterJ) [inline book/templating.rst]: If there is no PHP helper for it, it should be added during this stabilisation phase imo. Symfony 2.x still has to support PHP templating for 100%

**Comment by WouterJ** [comment]
Thanks @javiereguiluz! I'm closing this one in favor of https://github.com/symfony/symfony-docs/pull/5574 , where I continued your work here.

**Pull request closed without merging**